### PR TITLE
refactor: LNURLPay & MakeInvoice should use the new accessible TextField

### PR DESCRIPTION
**Pull request by dylancom**
Currently labels aren't clickable in those screens and styles are inconsistent.
This makes also the rendering more clear which became too abstract within methods.

### CODE REVIEW

File: src/app/screens/LNURLPay.tsx [hunk 1]
```diff
@@ -28,6 +28,14 @@ type Props = {
   origin?: Origin;
 };
 
+const Dt = ({ children }: { children: React.ReactNode }) => (
```
Comment (escapedcat) [inline src/app/screens/LNURLPay.tsx:31]: nit: We do this with `Button` as well, but I feel like naming these as the HTML-tag might be somewhat confusing.  
No need to tackle it here and now. Just a thought.

Comment (escapedcat) [reply src/app/screens/LNURLPay.tsx:31]: Might just be a react-coding-style-thing I'm not used to though.

Comment (dylancom) [reply src/app/screens/LNURLPay.tsx:31]: It's what I'm used to by known UI libs for e.g.: https://mui.com/components/buttons/
But debatable for sure :)

Comment (escapedcat) [reply src/app/screens/LNURLPay.tsx:31]: No worries, thanks for the link. Might be worth looking at some libs.

**Review by bumi — APPROVED** [review]
utACK

**Pull request merged**